### ISSUE TITLE
[BUG]Edit in table file trigger rerender in embedded file

### ISSUE THREAD

**Issue by Quorafind** [trey-wallis/obsidian-notion-like-tables#541]
### Describe the bug

I can't select a new type for column when I open the table file and also embed it into a note. And both the note and file is opened.

https://github.com/trey-wallis/obsidian-notion-like-tables/assets/13215013/b389f745-347c-4ac0-a56a-fa8d656c8633

### Steps to reproduce

1. Create a new table file.
2. Embed it into note.
3. Open this table file and also the note.
4. Click on any column Thead.
5. See error.

### Expected behavior

Edit on table file should not cause other files rerender.

### Are you using the mobile app?

No

### Obsidian debug info

```
SYSTEM INFO:
	Obsidian version: v1.3.6
	Installer version: v1.2.7
	Operating system: Windows 10 Pro for Workstations 10.0.19045
	Login status: logged in
	Catalyst license: vip
	Insider build toggle: on
	Live preview: on
	Legacy editor: off
	Base theme: light
	Community theme: Minimal v6.3.3
	Snippets enabled: 11
	Restricted mode: off
	Plugins installed: 67
	Plugins enabled: 66
		1: Longform v2.0.3
		2: Obsidian Memos v1.9.7
		3: Obsidian42 - BRAT v0.7.0
		4: Pandoc Plugin v0.4.1
		5: QuickAdd v1.0.2
		6: QuickShare v1.3.1
		7: Readwise Official v2.0.1
		8: Pane Relief v0.5.0
		9: Quick Explorer v0.2.8
		10: Paste image rename v1.6.1
		11: Outliner v4.7.1
		12: Projects v1.15.3
		13: Privacy Glasses v0.7.1
		14: Hover Editor v0.11.11
		15: Icon Folder v2.1.2
		16: Lapel v0.1.1
		17: Link Favicons v1.8.1
		18: Double Click Tab v1.1.2
		19: Export Image plugin v1.0.4
		20: Footnote Shortcut v0.1.2
		21: Commander v0.5.0
		22: Another Quick Switcher v8.12.0
		23: Better Command Palette v0.15.0
		24: Plugin Groups v2.1.0
		25: Canvas Mindmap v0.1.3
		26: Canvas Block Reference v0.0.4
		27: Latex Suite v1.7.2
		28: Link Nodes In Canvas v0.0.5
		29: Icon Shortcodes v0.9.7
		30: Auto Link Title v1.4.1
		31: Better Word Count v0.9.6
		32: Kanban v1.5.3
		33: Floating Search v3.3.1
		34: Gemmy v0.0.3
		35: Zoom v1.1.2
		36: Word Splitting for Simplified Chinese in Edit Mode and Vim Mode v1.10.0
		37: Webpage HTML Export v1.6.0
		38: WakaTime v1.0.0
		39: ToggleList v1.2.0
		40: Templater v1.16.0
		41: Tasks v3.10.0
		42: Task Progress Bar v1.6.1
		43: Tag Wrangler v0.5.11
		44: Table Enhancer v0.4.1
		45: Tab Switcher v0.0.2
		46: Surfing v0.8.16
		47: Style Settings v1.0.6
		48: Obsidian42 - Strange New Worlds (SNW) v1.2.2
		49: Natural Language Dates v0.6.1
		50: Linter v1.15.2
		51: Hider v1.2.4
		52: Easy Typing v5.1.9
		53: Excalidraw v1.9.3
		54: Dataview v0.5.56
		55: Vault name in status bar v1.0.1
		56: Dynamic Timetable v1.4.0
		57: Copy document as HTML v0.4.3
		58: Awesome Image v0.1.2
		59: Smart Rename v1.0.1
		60: Advanced Paste v2.4.0
		61: Tasks Calendar Wrapper v0.1.16
		62: Better Default Setting v0.0.1
		63: Surfing Key v0.0.1
		64: Notion-Like Tables v6.17.0
		65: Banners v1.3.3
		66: Dynamic Highlights v0.3.1

RECOMMENDATIONS:
	Custom theme and snippets: for cosmetic issues, please first try updating your theme and disabling your snippets. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.
```

**Comment by trey-wallis** [comment]
@Quorafind Thanks for this report. This is something I hadn't tested. Will fix today.

**Comment by trey-wallis** [comment]
@Quorafind Fixed in 6.18.0
